### ISSUE TITLE
[Feature Request] Add per webapp audio entry

### ISSUE THREAD

**Issue by Nathoufresh** [linuxmint/webapp-manager#178]
Would it be possible to have the right name and icon of the webapp in the audio menus? At the moment there are multiple entries for (let's say) firefox so it is not straight forward to see which firefox entry match to which webapp.

**Comment by micahcochran** [comment]
I was able to configure TIDAL, a Spotify competitor, to appear in the "Sound & Video" category under the Start Menu.  The find icons button is nice, but the icon for it is not not intuitive.  I completely missed it.

Are you asking for this portion of the interface to be updated so it reports the Category in the table?  So that TIDAL would have a Category column with "Sound & Video".
![image](https://user-images.githubusercontent.com/7433104/172987658-a32d4b7e-19a2-4242-a6f2-2c5b5fb30faa.png)

Please confirm or clarify.  This was my best guess.

**Comment by Nathoufresh** [comment]
No. Actually I'm talking about pulseaudio entries. So at the moment if I play a video with a YouTube WebApp, the audio entry in the pulseaudio applet is Firefox. So when you have multiples WebApps that are playing sound you cannot differentiate them.

![image](https://user-images.githubusercontent.com/39437515/173027665-5e0915c9-b3b9-4051-9e19-71044cf1fdde.png)
